<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt_test2Class</class>
 <widget class="QWidget" name="Qt_test2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt_test2</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>601</width>
     <height>181</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>160</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>x:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>y:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="text">
         <string>1.5</string>
        </property>
        <property name="placeholderText">
         <string>1.0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string>1.5</string>
        </property>
        <property name="placeholderText">
         <string>1.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>160</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="LINEAR">
        <property name="text">
         <string>LINEAR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="NEAREST">
        <property name="text">
         <string>NEAREST</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AREA">
        <property name="text">
         <string>AREA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LANCZOS4">
        <property name="text">
         <string>LANCZOS4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CUBIC">
        <property name="text">
         <string>CUBIC</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>171</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>なんか追加する用</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>601</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="filename">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>ここに画像ファイルをドラッグアンドドロップ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ファイル名：</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>解像度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="resolutionX">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="resolutionY">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>601</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <zorder>line</zorder>
  <zorder>tabWidget</zorder>
  <zorder>frame</zorder>
  <zorder>save</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_test2.qrc"/>
 </resources>
 <connections/>
</ui>
